DOCKER.md

```diff
@@ -25,19 +25,19 @@ you and your users can direct s3 server requests to example.com.
 docker run -d --name s3server -p 8000:8000 -e HOST_NAME=new.host.com scality/s3server
 ```
 
-#### ACCESS_KEY and SECRET_KEY
```

Comment:
You can add a note at the bottom that the old `ACCESS_KEY` and `SECRET_KEY` environment variables are now deprecated.